src/components/elements/back-to-top.tsx

```diff
@@ -15,20 +15,26 @@ const BackToTop = () => {
 
   useEventListener("scroll", useDebounceCallback(onScroll, 200))
 
+  const handleClick = () => {
+    scrollTo({
+      left: 0,
+      top: 0,
+      behavior: !!window.matchMedia("(prefers-reduced-motion: reduce)")?.matches ? "instant" : "smooth",
+    })
+    const skipLink = document.querySelector(".skiplink") as HTMLElement
+    if (skipLink) {
+      skipLink.focus()
+    }
```

Comment:
What about going to `#main-content`? Probably also want to prevent scroll on the focus since we're doing that separately.